distribution/lib/Standard/Table/0.0.0-dev/src/Data/Type/Value_Type.enso

```diff
@@ -186,23 +186,97 @@ type Value_Type
         Value_Type.Date_Time _ -> True
         _ -> False
 
+    ## PRIVATE
+       Specifies if values of the given type can be compared for ordering.
+    has_ordering : Boolean
+    has_ordering self = case self of
+        Value_Type.Integer _ -> True
```

Comment:
Why not `Value_Type.Byte`?

Reply:
and I think `Boolean` has an order both in Enso and in DB (`False < True`)

Reply:
Byte - sure, I kind of thought of it as a binary format and not a number, but obviously that is wrong - I will amend.

As for Booleans - I remember a discussion long time ago that ordering booleans does not make sense so Enso should not support this. But I think the decision has been changed since and indeed, Booleans are comparable in Enso now. Then I can add them here as well.